Pear/Controller/HomeViewController.swift

```diff
@@ -355,4 +357,19 @@ extension HomeViewController: BlockDelegate {
     
 }
 
-
+extension HomeViewController: UserAgreementDelegate {
+    func setupUserLicenseAgreementView() {
```

Comment:
add newline before function

Reply:
I think the style guide doesn't have a newline though, is this preferable?